website/docs/blueprints/data-analytics/binpacking-custom-scheduler-eks.md

```diff
@@ -0,0 +1,56 @@
+---
+sidebar_position: 6
+sidebar_label: Binpacking for Amazon EKS
+---
+
+
+# Binpacking for Amazon EKS
+
+## Introduction
```

Comment:
Let's talk little bit more about the problem statement and what is "LeastAllocated" and "MostAllocated". It would be nice to show the difference in images if you have any.

Reply:
done